Sources/swift-memcache-gsoc-example/main.swift

```diff
@@ -0,0 +1,50 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the swift-memcache-gsoc open source project
+//
+// Copyright (c) 2023 Apple Inc. and the swift-memcache-gsoc project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-memcache-gsoc project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIOCore
+import NIOPosix
+import SwiftMemcache
+
+// Create an event loop group with a single thread
+let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+// Gracefully shutdown the event loop group when it's no longer needed
+defer {
+    try! group.syncShutdownGracefully()
+}
+
```

Comment:
Can we please use the `@main` entry point instead of top level code. This could look like the following

```
@main
struct Program {
    static func main() async throws {
        // Your code goes here
    }
}
```

Importantly you have to rename your `main.swift` file to something like `Program.swift`